spec/factories/tasks.rb

```diff
@@ -5,5 +5,8 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+      trait :done_task do
+        status { 2 }
```

Comment:
2だと何を意味するのか分からないので、Symbolで記載して伝わるようにしましょう！
`マジックナンバー`で調べてみてください。
```suggestion
        status { :done }
```

Reply:
d4bae1d で修正しました。